tests/framework/PostProcessors/BasicStatistics/test_time_dep_meta.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" ?>
```

Comment:
Any chance we could trim this case to a minimal necessary test case? This is a lot to comb through.